src/symbolics.jl

```diff
@@ -97,7 +97,10 @@ const default_symbolics_constants =
         ::Type{Num},
         x::SBML.Math;
         mapping = default_symbolics_mapping,
-        convert_time = (x::SBML.MathTime) -> Num(Variable(Symbol(x.id))).val,
+        convert_time = function (x::SBML.MathTime)
+            sym = Symbol(x.id)
+            Symbolics.unwrap(first(@variables \$sym))
```

Comment:
```
help?> Symbolics.unwrap
  No documentation found.

  Symbolics.unwrap is a Function.
```

what does this actually do?

Reply:
it changes from Num => Sym. `.val` technically works, but it's not a safe way to do it (edge cases like symbolic arrays).

Reply:
it should probably get a docstring, agreed (@shashi)

Reply:
OK I'm wondering why we used `.val` in the first place.

Anyway this looks good to go, I'll just need to find a bit of time to get through the possibilities, ideally killing the macro from there :D

Expect merged in a few days.